src/Blockcore.Explorer/ClientApp/src/app/explorer/contracttransaction/contracttransaction.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, HostBinding, OnInit, OnDestroy, HostListener } from '@angular/core';
+import { ActivatedRoute, Route, Router } from '@angular/router';
+import { ApiComponent } from 'src/app/api/api.component';
+import { ApiService } from 'src/app/services/api.service';
+import { SetupService } from 'src/app/services/setup.service';
+
+@Component({
+   selector: 'app-contracttransaction-component',
+   templateUrl: './contracttransaction.component.html'
+})
+export class ContractTransactionComponent implements OnInit, OnDestroy {
+  @HostBinding('class.content-centered-top') hostClass = true;
+
+  info: any;
+  node: any;
+  blockchain: any;
+  network: any;
+  configuration: any;
+  consensus: any;
+  peers: any;
+  blocks: any;
+  transactions: any;
+
+  timerInfo: any;
+  timerBlocks: any;
+  timerTransactions: any;
+  transaction: any;
+  detailsVisible = false;
+  lastBlockHeight: number;
+  subscription: any;
+  error: Error;
+
+  constructor(
+    private api: ApiService,
+    private router: Router,
+    public setup: SetupService,
+    private activatedRoute: ActivatedRoute) {
+
+    this.activatedRoute.paramMap.subscribe(async params => {
```

Comment:
This is a subscription that should be removed in the ngOnDestroy I think, to avoid leaks?